src/hooks/useFollowScroll.ts

```
@@ -0,0 +1,66 @@
+      currentPosition.current += distance * damping;
+      elementRef.current.style.transform = `translateY(${currentPosition.current - initialPosition}px)`;
+
+      if (Math.abs(distance) > 0.5) {
+        requestAnimationFrame(updatePosition);
+      }
+    }
+  }, [damping, initialPosition]);
+
+  useEffect(() => {
+    if (deviceSize !== "desktop") {
+      elementRef.current && (elementRef.current.style.transform = "translateY(0)");
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)